### DISCUSSION TITLE
API Err meaning on the jetstream report of nats-box

### DISCUSSION THREAD

**Discussion by albionb96** [nats-io/nats-server#5051] [Q&A] [answered]
Hi dear community,

I had a question regarding the API Err parameter that I see when calling the Jetstream report on the nats-box, using this command:
`nats server report jetstream --user <<user>> --password <<pass>>`

I see sometimes too many Errors on this parameter like this: 
![image](https://github.com/nats-io/nats-server/assets/49485798/7587a902-41e0-47fb-8470-cf5e8d670b2b)

although everything seems to work just fine.

Thank you very much in advance.

**Comment by ripienaar** [comment, marked as answer]
Each server serves the JetStream API, any failure to serve API calls will increment this, cases like:

 * Asking for information for streams/consumers that dont exist
 * Sending invalid API requests
 * Resources not available to serve the request

And some other cases, unfortunately the simple counter is a bit course grained to be as useful

**Reply by albionb96** [reply]
Thank you very much.

It is interesting that this counter doesn't reset. I found out that this number of API Err much earlier happened and is not incrementing anymore.
